### DISCUSSION TITLE
redirect https problem

### DISCUSSION THREAD

**Discussion by flashboss** [community/community#35900] [Other Feature Feedback, Questions, & Ideas]
I wish my domain didn't use https redirect. Following the documentation I went to the ghpages configuration of my application A, I created a custom domain B and unchecked enforcing https redirect. When I execute: http: // B I always get redirected to https: // B. What is missing?
<img width="785" alt="Schermata 2022-10-12 alle 13 28 37" src="https://user-images.githubusercontent.com/473269/195333122-dd582cb3-4c32-4cf8-87ed-9795d82dd6b7.png">

**Comment by yoannchaudet** [comment]
There is no redirection happening when accessing the URL over HTTP only:

<img width="594" alt="image" src="https://user-images.githubusercontent.com/14911070/195466860-9d273409-edb7-4175-a806-61242f89643b.png">

The redirection you are seeing happening in your browser is caused by our HSTS header:

<img width="547" alt="image" src="https://user-images.githubusercontent.com/14911070/195467045-5d6f11c2-295e-426a-87f9-5f6d28c36d65.png">

If you access the HTTPS URL once, your browser will remember the HTTPS preference.

**Comment by flashboss** [comment]
thanks you!
